src/widgets/PieChart.js

```diff
@@ -0,0 +1,116 @@
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2018
+ */
+
+import React, { useState, useEffect } from 'react';
+import { View, StyleSheet } from 'react-native';
+import PropTypes from 'prop-types';
+import { VictoryLabel, VictoryPie } from 'victory-native';
+import { APP_FONT_FAMILY, GREY } from '../globalStyles';
+
+/**
+ * A charting widget for displaying Pie chart report.
```

Comment:
Docstrings imo should be a summary to help other programs HOW to use something, rather than WHAT something is. If I am reading this file, probably I want to use your new `PieChart` component.. then I already really know what this is going to be, I just don't know how to use it yet!

Reply:
Agree, we should be more explicit here. Anyway I was planning to don't touch it now and create an issue to update all docstrings later, given instructions after having a bigger picture of what to write that can be really useful! :-)

Reply:
100% agree, better to avoid confusions. Issue added here #1745